### ISSUE TITLE
⌨ VoiceOver does not announce error

### ISSUE THREAD

**Issue by brindasasi** [GCTC-NTGC/gc-digital-talent#14197]
### ⌨ Accessibility and Expected Behaviour

Voice over does not say what is wrong in this situation. It is simply doesn't save the form.
I believe error summary alert is not showing up in this scenario and hence there is no announcement for that. 

#### Screenshot

<img width="764" height="817" alt="Image" src="https://github.com/user-attachments/assets/89ca532f-830e-4f08-83a8-fd11547311f3" />

### 📋 Steps to Reproduce

1. Turn on VoiceOver
2. Go to Applicant Dashboard 
3. Go to http://localhost:8000/en/applicant/personal-information#language-section
4. Edit the section as per the screen shot and leave out first official language
5. Try save changes.
6. Observe nothing happens and you don't hear any announcement that there is an error 

### What operating systems are you encountering the problem on?

macOS

### What browsers are you encountering the problem on?

Chrome

### What assistive technologies are you encountering the problem on?

VoiceOver

### ✅ Acceptance Criteria

- [ ] Error alert should be shown and error message should be announced for the missing field
